run_cloc_azure.sh

```diff
@@ -19,9 +19,12 @@ echo "================Starting experiment for input: $line"
 slugFull=$(echo ${line} | cut -d',' -f1)
 slug=$(echo ${line} | cut -d',' -f1 | rev | cut -d'/' -f1-2 | rev)
 sha=$(echo ${line} | cut -d',' -f2)
+modifiedslug=$(echo ${slug} | sed 's;/;.;' | tr '[:upper:]' '[:lower:]')
+short_sha=${sha:0:7}
+modifiedslug_with_sha="${modifiedslug}-${short_sha}"
 
 echo "================Installing the project"
-MVNOPTIONS="-fn -Ddependency-check.skip=true -Dmaven.repo.local=$AZ_BATCH_TASK_WORKING_DIR/$input_container/dependencies -Dgpg.skip=true -DfailIfNoTests=false -Dskip.installnodenpm -Dskip.npm -Dskip.yarn -Dlicense.skip -Dcheckstyle.skip -Drat.skip -Denforcer.skip -Danimal.sniffer.skip -Dmaven.javadoc.skip -Dfindbugs.skip -Dwarbucks.skip -Dmodernizer.skip -Dimpsort.skip -Dmdep.analyze.skip -Dpgpverify.skip -Dxml.skip"
+MVNOPTIONS="-fn -Ddependency-check.skip=true -Dmaven.repo.local=$AZ_BATCH_TASK_WORKING_DIR/dependencies_${modifiedslug_with_sha} -Dgpg.skip=true -DfailIfNoTests=false -Dskip.installnodenpm -Dskip.npm -Dskip.yarn -Dlicense.skip -Dcheckstyle.skip -Drat.skip -Denforcer.skip -Danimal.sniffer.skip -Dmaven.javadoc.skip -Dfindbugs.skip -Dwarbucks.skip -Dmodernizer.skip -Dimpsort.skip -Dmdep.analyze.skip -Dpgpverify.skip -Dxml.skip"
 git clone https://github.com/$slug $slug
```

Comment:
In the installation part, 
elif [[ "$slug" == "openpojo/openpojo" ]]; then
this ifs are different between intall-project.sh and this file.

Reply:
Keep the current openpojo in install-project.sh and copy and comment out the commands under this scripts' openpojo to be in install-project.sh in case we need it in the future.

Thanks for checking if clauses for installing projects. Was openpojo the only one that differed?

Reply:
It seems like it is the only one, "apache/incubator-dubbo" is the same in both files If I am not wrong